x-pack/plugins/security_solution/public/timelines/components/timeline/body/events/stateful_event.tsx

```diff
@@ -78,7 +79,7 @@ const StatefulEventComponent: React.FC<Props> = ({
   const dispatch = useDispatch();
   const [showNotes, setShowNotes] = useState<{ [eventId: string]: boolean }>({});
   const expandedEvent = useDeepEqualSelector(
-    (state) => state.timeline.timelineById[timelineId].expandedEvent
+    (state) => (state.timeline.timelineById[timelineId] ?? timelineDefaults).expandedEvent
```

Comment:
the selector should be me memoized too like
https://github.com/angorayc/kibana/blob/alerts-details/x-pack/plugins/security_solution/public/timelines/components/flyout/header/index.tsx#L124-L127